scripts/mppc_to_multiprobe.py

```diff
@@ -194,48 +203,50 @@ def get_mppc_rotation(images):
     delta_x_A1_H1 = spotcenter_H1[1] - spotcenter_A1[1]
     angle = numpy.arctan2(delta_y_A1_H1, delta_x_A1_H1)  # [rad]
-    mppc_rotation = -math.degrees(angle)  # convert to degree and change sign for correcting the rotation
+    mppc_rotation = -angle  # change sign for correcting the rotation
 
     # control
     delta_x_A8_H8 = spotcenter_H8[1] - spotcenter_A8[1]
     angle_control = numpy.arctan2(delta_y_A8_H8, delta_x_A8_H8)  # [rad]
-    angle_control_degree = -math.degrees(angle_control)  # convert to degree and change sign for correcting the rotation
+    angle_control_degree = -angle_control  # change sign for correcting the rotation
 
-    # check that angles are the same, allow a 1 degree margin
+    # check that angles are the same, allow a 0.5 degree margin
     if not almost_equal(mppc_rotation, angle_control_degree, atol=0.5):
         raise ValueError("The angle found based on the images of cell A1 and H1, should be the same angle"
                          "as for cells A8 and H8. However, the angle of A1/H1 is {:.2f} "
                          "whereas the angle of A8/H8 is {:.2f}.".format(mppc_rotation, angle_control_degree))
 
-    return mppc_rotation  # angle
+    return mppc_rotation  # angle in rad
 
 
 def get_mppc_offset(image, phi):
     """
     Calculates the offset of the MPPC detector in space. MPPC detector should be centered in regard to the scanning
     beamlet, when in single beam mode.
     Calculates the distance of the active MPPC cell image (bright square) in regard to the center of the cell image.
-    Note: Return value is based on top/left corner (0,0).
-    :param image: (ndarray) Preprocessed E4 cell images.
-    :param phi: (float) Rotation angle to transform the mppc image coordinate system to the multiprobe
+    :param image: (ndarray, dtype: int64) Preprocessed E4 cell images. Shape is (y, x).
+    :param phi: (float) Rotation angle in degrees to transform the mppc image coordinate system to the multiprobe
                 coordinate system.
-    :returns: (tuple of 2 float) The distance to the E4 cell image center in y and x in px.
+    :returns: (tuple of 2 float) The distance to the E4 cell image center in x and y in px.
+              Return value is based on top/left corner (0,0).
     """
 
     rot_matrix = numpy.array(((numpy.cos(phi), - numpy.sin(phi)), (numpy.sin(phi), numpy.cos(phi))))
 
-    # calculate center of mass (y, x) of the bright spot
-    spotcenter_E4 = center_of_mass(image)
+    # calculate center of mass (x, y) of the bright spot
+    spotcenter_E4 = center_of_mass(image)[::-1]  # invert to follow convention (x, y) in px
     spotcenter_E4_rot = numpy.dot(spotcenter_E4, rot_matrix)  # coordinate transform into system of multiprobe
 
     # center of image
-    image_center = (image.shape[0]/2., image.shape[1]/2.)  # (y, x) [px] is at 400 x 400
+    image_center = (image.shape[0]/2., image.shape[1]/2.)[::-1]  # (x, y) in px is at 400 x 400
```

Comment:
You can simply write `image_center = image.shape[1] / 2, image.shape[0] / 2`

Reply:
I didn't do it to have it similar to the calculation of the spotcenter, but changed it now as suggested.